### PR TITLE
File added: aboutme.txt and html assignment

### PR DESCRIPTION
**Pull request by ajayk98**

### CODE REVIEW

File: ajay/HTML/formtable.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+		<p>1)Self Adaptive access control and delegation in cloud computing for online shopping system<br>
+		<h4 style="color:red;">Internship:</h4>
+		<p>1)Starting intership in Amber softTech System</br></p>
+		<h3  style="color:black;">Interest:</h3>
+		<img src="/home/ajay/Desktop/MI-6-MEAN/ajay/HTML/Images/inter.png"alt="sinhgad college">
+		<p>I like suffring internet ,playing  cricket,reading books.</p>
+		<h1>Education :</h2>
+<table style="width:50%">
+	<tr>
+		<th colspan="2">Institution</th>
```
Comment (poojakamble) [inline ajay/HTML/formtable.html:33]: remove Extra space

File: ajay/HTML/formtable.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+		 <h3 style="color:red";>Stuff I need to do ASAP</h3>
+   	 <ul>
+      	  	 <li>To learn HTML</li>
+      		  <li>To play cricket</li>
+       		  <li>To watch movie</li>
+		  <li>To do lunch</li>
+		  <li>To read book</li>
+    	 </ul>
+	     <Form>
+				<h3>Contact me</h3>
```
Comment (poojakamble) [inline ajay/HTML/formtable.html:75]: remove extra identation

File: ajay/HTML/formtable.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+		<td> <a href="https://www.icbse.com/schools/venkatesh-madhymik-h-s-o14e4z">Shri Venkatesh Secondary And Higher Secondary School,Latur</a> </td>
+		<td><img src="/home/ajay/Desktop/MI-6-MEAN/ajay/HTML/Images/IMG_20200429_142525.jpg"alt="school latur"  width="90px" height="90px"></td>    
+		<td>2012</td>
+		<td>SSC</td>    
+	</tr>
+</table>
+		<h2>FIND ME ON THE WEB</h2>
+		<h2 style="color:blue">MY TODO LIST</h2>
+		<h3 style="color:red";>Stuff I need to do somedays</h3>
+  	 <ol>
```
Comment (poojakamble) [inline ajay/HTML/formtable.html:59]: one tab and space

File: ajay/HTML/formtable.html [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+      table, th, td
+      {
+      border: 1px solid black;
+      }
+      th, td 
+      {
+      padding: 5px;
+      }
+   </style>
+</head>
```
Comment (JafarPathan) [inline ajay/HTML/formtable.html:14]: use indentation

File: ajay/HTML/formtable.html [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+   <h4 style="color:red;">Final year Project :</h4>
+   <p>1)Self Adaptive access control and delegation in cloud computing for online shopping system<br>
+   <h4 style="color:red;">Internship:</h4>
+   <p>1)Starting intership in Amber softTech System</br></p>
+   <h3  style="color:black;">Interest:</h3>
+   <img src="/home/ajay/Desktop/MI-6-MEAN/ajay/HTML/Images/inter.png"alt="sinhgad college">
+   <p>I like suffring internet ,playing  cricket,reading books.</p>
+   <h1>
+   Education :</h2>
+   <table style="width:50%">
```
Comment (JafarPathan) [inline ajay/HTML/formtable.html:34]: h1 or h2 .....?

File: ajay/HTML/formtable.html [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+      <h1 style="padding-top:15px">Ajay Kshirsagar</h1>
+      <span style="color:blue">kshirsagarajay09@gmail.com</span>
+   </header>
+   <h2  style="color:black;">About Me :</h2>
+   <p> I am Ajay Kshirsagar kamlakar. I Completed my engineering in smt kashibai navale college of engineering in Pune.
+      In Information technology in 2018.
+   </p>
+   <h3 style="color:black;">Work:</h3>
+   <h4 style="color:red;">Final year Project :</h4>
+   <p>1)Self Adaptive access control and delegation in cloud computing for online shopping system<br>
```
Comment (JafarPathan) [inline ajay/HTML/formtable.html:26]: missing p tag

File: ajay/HTML/formtable.html [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+   <h4 style="color:red;">Internship:</h4>
+   <p>1)Starting intership in Amber softTech System</br></p>
```
Comment (JafarPathan) [inline ajay/HTML/formtable.html:28]: you used br tag before p tag

File: ajay/HTML/formtable.html [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+   <input type="num" id="contact" name="phone no" value=""><br><br>
+   <label for="message">Message:</label>
+   <textarea name="message" rows="10" cols="30">
+   Message send.
+   </textarea><br><br>
+   <input type="submit" value="Send your Message" style="color:blue">
+   </fieldset>
+   </form>
+</body>
+</html>
```
Comment (JafarPathan) [inline ajay/HTML/formtable.html:97]: use indentation

File: ajay/HTML/formtable.html [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+      <fieldset>
+         <legend >Contact </legend>
+         <br />
+   <form action="/action_page.php"> 
+   <label for="Name"> Name:</label>
+   <input type="text" id="fname" name="fname" value=""><br><br>
+   <label for="email">Email:</label>
+   <input type="text" id="emial" name="email" value=""><br><br>
+   <label for="phoneno"> Mobile :</label>
+   <input type="num" id="contact" name="phone no" value=""><br><br>
```
Comment (JafarPathan) [inline ajay/HTML/formtable.html:88]: type="num".......?

**Pull request merged**